### DISCUSSION TITLE
Hi does anyone if it is possible to backup the Budibase DB?

### DISCUSSION THREAD

**Discussion by starsang1995** [Budibase/budibase#10615] [Help]
Hi we are considering using Budibase DB in Budibase for some of our apps. We haven't used it before because we don't know if we can create backups for Budibase DB so that we are not afraid of losing data if something goes wrong with our Budibase instance. Does anyone know if it is possible to backup Budibase DBs? Many thanks ahead!

**Comment by ConorWebb96** [comment]
Hey @apsonandtesla,

Are you cloud based or self-hosted?

From memory simply exporting the app will also export the rows of your tables within the Budibase DB. Just make sure that the exclude rows toggle isn't toggled on.
<img width="410" alt="Screenshot 2023-05-17 at 08 43 50" src="https://github.com/Budibase/budibase/assets/126772285/7ab1d1c7-fc06-4150-ac75-aee27bed01f0">

There is also an inbuilt paid feature called backups which runs and backups an app every time it is published. You can also click the `Create new backup` button in order to backup everything manually.
<img width="852" alt="Screenshot 2023-05-17 at 08 45 28" src="https://github.com/Budibase/budibase/assets/126772285/36dc4caf-0ab5-4e80-be00-928ce9bde6c4">

Lastly if you are using the cli, there is an inbuilt backup function which will backup the entire instance. Linked provided below to relevant docs.
https://docs.budibase.com/docs/budibase-cli-reference#commands-2

I hope this is helpful!

**Comment by sghriyadh** [comment]
is it not possible to give self hosting client a simple monthly tier of $3-5 per month for Automatic daily Backup of all the apps at a specific time of day( like a cron job builtin).

Give some relief to personal tier and create Plus tier at $3-$5/month on self hosted budibase.

**Comment by melohagan** [comment]
There's also more on backups here: https://docs.budibase.com/docs/backups-1
